source/scpd/types/Stellar_SCP.d

```diff
@@ -76,14 +76,14 @@ struct SCPStatement {
 
```

Comment:
The commit message should be fixed. It's not about envelope signing, it's about **confirmation statement** signing. The envelopes themselves already contain a signature using a random R unrelated to the `value_sig` here.

Reply:
And again, the commit message should explain why the change was made.